### DISCUSSION TITLE
求UZI跳枪课程一份。。。

### DISCUSSION THREAD

**Discussion by qifeng4567** [share121/inter-knot#3202] [其他]
求UZI跳枪课程一份  没有就算了。。。
![Uploading 3bad93ec010759ffdaa733864201f7fa.jpeg…]()

**Comment by StrinovaMaster** [comment]
https://github.com/user-attachments/assets/ead50e32-b2e1-43ce-8fa9-63e78b9e5735

这里有uzi跳枪课程

**Reply by qifeng4567** [reply]
666你人还怪好嘞
